test/ConsoleAgent.js

```diff
@@ -27,13 +27,11 @@ describe('ConsoleAgent', function() {
 
   it('should translate objects', function(done) {
     frontendClient.once('Console.messageAdded', function(message) {
-      // expect(message.message.parameters).to.deep.equal([{
-      //   type: 'object',
-      //   subtype: undefined,
-      //   objectId: 'console:1:1',
-      //   className: 'Object',
-      //   description: 'Object'
-      // }]);
+      var parameters = message.message.parameters[0];
+      expect(parameters.type).to.equal('object');
+      expect(parameters.subtype).to.equal(undefined);
+      expect(parameters.className).to.equal('Object');
+      expect(parameters.description).to.equal('Object');
```

Comment:
FWIW, it may be enough to replace `.deep.equal` with `.deep.include` or `.deep.contains` + remove `objectId` from the expected value.